learningobjects/models.py

```diff
@@ -32,12 +32,12 @@ class LearningObject(models.Model):
     path_xml = models.TextField()
     mpath = models.TextField()
```

Comment:
If so, I think we should include it here.  I imagine the migration to using MPTT is going to be pretty nasty, but just a guess, and we can always just reset migrations if it proves to problematic.